alleyinteractive/wp-newsletter-builder
# Issue-13: convert templates/layouts from static files to custom post type

### Description

This PR converts templates/layouts from static files to a custom post type. It updates all references to the static files and file names to use the new custom post type, pulling from the REST API or WP_Query as appropriate.

### Changes
- Created a custom post type for templates/layouts
- Updated all references to static files and file names to use the new custom post type
- Replaced static file storage with REST API or WP_Query calls

### Testing
1. Create or update a template/layout through the admin interface
2. Verify that the new custom post type is used for storing and retrieving templates/layouts

### Related Issue
- Issue #13: [convert templates/layouts from static files to custom post type](https://github.com/alleyinteractive/wp-newsletter-builder/issues/13)

### Notes
- This PR assumes that the wp-newsletter-builder plugin is properly set up and the required custom post type has been registered.
- Please make sure to thoroughly test the changes before merging, as this is a significant change to the template/layout handling system.
